systemtests/src/test/java/io/enmasse/systemtest/common/catalog/ServiceCatalogWebTest.java

```diff
@@ -87,6 +88,7 @@ void testProvisionAddressSpaceStandard() throws Exception {
         provisionedServices.put(namespace, standard);
         OpenshiftWebPage ocPage = new OpenshiftWebPage(selenium, addressApiClient, getOCConsoleRoute(), developer);
         ocPage.openOpenshiftPage();
+        ocPage.checkReachableWebPage();
```

Comment:
openOpenshiftPage() already contains execution of checkReachableWebPage() method, so this is not necessary. This additional check can be also removed from next tests (in commits bellow)

Reply:
damn I forgot to remove them before commiting! removed now